nuxt.config.js

```diff
@@ -32,6 +44,9 @@ export default defineNuxtConfig({
     },
   },
 
+  build: {
+    transpile: ["vuetify"],
+  },
   vue: {
     compilerOptions: {
       isCustomElement: (tag) => ["md-linedivider"].includes(tag),
```

Comment:
What is that?

Reply:
Vue doesn't know when a component is local or from vuetify